portfolio/src/main/webapp/script.js

```diff
@@ -13,17 +13,23 @@
 // limitations under the License.
 
 function displayComment(limValue){
-    fetch('/comments?limit='+ limValue).then(response => response.json()).then(data => {
-        const commentContainer = document.getElementById("comment_container");
+    var fetchString = '/comments';
+    if(limValue){
+        fetchString += "?limit=";
+        fetchString += limValue;
+    }
```

Comment:
this is great! is it being used by the api endpoint though? I don't see /comments being updated to handle this in this PR. If this is coming in another PR, i would suggest moving this change to that PR.

Reply:
This change is actually from a former PR and its just a restructuring of something that was there before. Should I make these their own pull requests in the future?

Reply:
ah gotcha. it's fine for this PR but unrelated changes should probably be in their own CL. I do now see how this is only a slight modification of an existing feature, but this probably would have been best as its own PR titled "limValue API parameter optionally passed via javascript" or something similar.

Reply:
ok! ill make a new PR request next time